src/main/kotlin/day03/Hero.kt

```
@@ -1,26 +1,24 @@
-    open var name: String,
-    open var hp: Int = 50,
-    open var mp: Int = 10,
+    open val name: String,
+    open var hp: Int = MAX_HP,
+    var sword: Sword? = null,
 ) {
+    companion object {
+        const val MOENY = 1000
+    }
 
     fun attack() {}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)